{
	"candidate": "CVE-2022-42966",
	"description": "An exponential ReDoS (Regular Expression Denial of Service) can be triggered in the cleo PyPI package, when an attacker is able to supply arbitrary input to the Table.set_rows method",
	"priority": "medium",
	"cvss": {
		"nvd": " CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H [7.5 HIGH]"
	},
	"references": [
		"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-42966",
		"https://research.jfrog.com/vulnerabilities/cleo-redos-xray-257186/"
	],
	"public_date": "2022-11-09 20:15:00 UTC",
	"packages": {
		"bionic": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "DNE"
			}
		},
		"devel": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "needs-triage"
			}
		},
		"esm-apps/jammy": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "needs-triage"
			}
		},
		"focal": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "DNE"
			}
		},
		"jammy": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "needs-triage"
			}
		},
		"kinetic": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "ignored",
				"note": "end of life, was needs-triage"
			}
		},
		"lunar": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "needs-triage"
			}
		},
		"mantic": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "needs-triage"
			}
		},
		"trusty": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "ignored",
				"note": "end of standard support"
			}
		},
		"upstream": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "needs-triage"
			}
		},
		"xenial": {
			"python-cleo": {
				"name": "python-cleo",
				"status": "ignored",
				"note": "end of standard support"
			}
		}
	}
}
